firefox-ios/Client/Coordinators/SummarizeCoordinator.swift

```diff
@@ -122,7 +122,7 @@ class SummarizeCoordinator: BaseCoordinator {
             descriptionText: descriptionText,
             descriptionTextA11yId: AccessibilityIdentifiers.Summarizer.tosDescriptionText,
             linkButtonLabel: .Summarizer.ToSAlertLinkButtonLabel,
-            linkButtonURL: URL(string: "https://www.mozilla.com"),
+            linkButtonURL: SupportUtils.URLForTopic("summarize-pages-ios"),
```

Comment:
@FilippoZazzeroni It still gives a 404 because the content is not ready yet. But this is the final slug that will be used for the SUMO article.